asciidoctorj-core/src/main/resources/org/asciidoctor/internal/asciidoctorclass.rb

```diff
@@ -2,6 +2,9 @@ module AsciidoctorJ
     include_package 'org.asciidoctor'
     module Extensions
         include_package 'org.asciidoctor.extension'
+        # Treeprocessor was renamed in to TreeProcessor in https://github.com/asciidoctor/asciidoctor/commit/f1dd816ade9db457b899581841e4cf7b788aa26d
+        # This is necessary to run against both Asciidoctor 1.5.5 and 1.5.6
+        TreeProcessor = Treeprocessor
```

Comment:
Aha! Thanks for pointing that out. You are absolutely correct. We do need the alias there. That's a different namespace (and thus different alias) from the one in core.

You are correct that we may not need this special case for handling Java types in the future. But we'll have to test it. After all, Ruby is using the Java native type directly in this case.